courses/views/v1/__init__.py

```diff
@@ -55,7 +55,13 @@ class ProgramViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = (
         Program.objects.filter(live=True)
         .exclude(products=None)
-        .select_related("programpage", "externalprogrampage", "platform")
+        .select_related(
+            "programpage",
+            "externalprogrampage",
+            "platform",
+            "programpage__language",
+            "externalprogrampage__language",
+        )
         .prefetch_related(courses_prefetch, products_prefetch)
```

Comment:
Just noticed we need to select_related the course language for the program courses.